201-ospatching-extension-on-ubuntu/azuredeploy.json

```diff
@@ -98,17 +106,41 @@
         "description": "Start time of patching."
       }
     },
-    "category": {
+    "idleTestScript": {
       "type": "string",
-      "defaultValue": "ImportantAndRecommended",
+      "defaultValue": "",
       "metadata": {
-        "description": "Type of patches to install."
+        "description": "The uri of the idle test script"
+      }
+    },
+    "healthyTestScript": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "The uri of the healthy test script"
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "The name of storage account."
+      }
+    },
+    "storageAccountKey": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "The access key of storage account."
       }
     }
   },
   "variables": {
+    "location": "[parameters('location')]",
+    "vmSize": "Standard_D1",
     "imagePublisher": "Canonical",
     "imageOffer": "UbuntuServer",
+    "ubuntuOSVersion": "14.04.2-LTS",
     "OSDiskName": "osdiskforospatching",
     "nicName": "myVMNic",
```

Comment:
Please use a name that follows this scheme for these variables - templateScenarioResourceName